Task_objects/index.js

```diff
@@ -1,24 +1,49 @@
-const cat = {
-  name: "Bertie",
-  breed: "Cymric",
-  color: "white",
-  greeting: function () {
-    console.log("Meow!");
-  },
```

Comment:
Consider adding a semicolon after the console.log statement for consistency with JavaScript best practices.